configs/CSC-Lumi-HPECray-zen3-MI250X-Slingshot/auxiliary_software_files/packages.yaml

```diff
@@ -0,0 +1,17 @@
+# Copyright 2023 Lawrence Livermore National Security, LLC and other
+# Benchpark Project Developers. See the top-level COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: Apache-2.0
+
+packages:
+  all:
+    require: 'target=x86_64:'
+    variants: amdgpu_target=gfx90a
+  tar:
+    externals:
+    - spec: tar@1.30
+      prefix: /usr
+
```

Comment:
This should include a `cray-libsci` external entry, and should specify that `blas` and `lapack` are not buildable:

```
package:
  cray-libsci:
    externals:
    - spec: ...
      prefix: ...
  blas:
    buildable: false
  lapack:
    buildable: false
```

Reply:
A less critical issue is that it would be useful to add external entries for all the rocm components here: you can generate that quickly with `spack external find -t rocm`.

Reply:
This also needs an external entry for `cray-mpich`.